nomos-services/key-management-system/src/backend/preload.rs

```diff
@@ -1,127 +1,129 @@
-use std::collections::HashMap;
+#[cfg(test)]
+mod tests {
+    use std::collections::HashMap;
 
-use bytes::Bytes;
-use overwatch::DynError;
-use serde::{Deserialize, Serialize};
-use zeroize::ZeroizeOnDrop;
+    use bytes::Bytes;
+    use overwatch::DynError;
+    use rand::rngs::OsRng;
+    use serde::{Deserialize, Serialize};
+    use zeroize::ZeroizeOnDrop;
 
-use super::KMSBackend;
-use crate::{keys::ed25519::Ed25519Key, secure_key::SecuredKey, KMSOperator};
+    use crate::{
+        backend::KMSBackend, keys::ed25519::Ed25519Key, secure_key::SecuredKey, KMSOperator,
+    };
 
-pub struct PreloadKMSBackend {
```

Comment:
@youngjoon-lee Clippy was complaining this struct is never used, since indeed it was only tested in tests. So I moved it all inside the `tests` module, and we can pull it out of here once we integrate it into something. Are you ok with this?